clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -151,13 +161,34 @@ func (ste *simpleTestExecutor) ExecutePhase(ctx Context, phase *api.Phase) []err
 }
 
 // ExecuteObject executes single test object operation based on provided object configuration.
-func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32) []error {
+func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32, operation OperationType) []error {
+	var errList []error
+	template, err := config.ReadTemplate(object.ObjectTemplatePath)
+	if err != nil {
+		return []error{fmt.Errorf("reading template (%v) error: %v", object.ObjectTemplatePath, err)}
+	}
+	gvk := template.GroupVersionKind()
+
 	if namespace == "" {
 		// TODO: handle cluster level object
 	} else {
-		// TODO: handle namespaces level object
+		objName := fmt.Sprintf("%v-%d", object.Basename, replicaIndex)
+		switch operation {
+		case CREATE_OBJECT:
+			if err := ctx.GetFramework().CreateObject(namespace, objName, template); err != nil {
+				errList = append(errList, err)
+			}
+		case UPDATE_OBJECT:
+			// TODO: add hanlding for update operation
+		case DELETE_OBJECT:
+			if err := ctx.GetFramework().DeleteObject(gvk, namespace, objName); err != nil {
+				errList = append(errList, err)
+			}
+		default:
+			errList = append(errList, fmt.Errorf("Unsupported operation %v", operation))
```

Comment:
Please also try to include objName in the error msgs wherever relevant. Otherwise it's hard to trace back to particular operation.

Reply:
error massages changed.